src/nonlinear/room-edits/main-deck/quarantine-bay.s

```diff
@@ -0,0 +1,31 @@
+; Changes: 
+; - Always spawn Hornoad.
+; - Change Hornoad Palette to RoS themed
+; - Hornoad always drops Red X
+
+
+; Always spawn hornoad. This is the location of the CheckOnEvent_NavBeforeQuarantineBay function
+; which spawns the hornoad if the vanilla event counter == 1. Because events have changed, we can
+; override this function to always spawn the hornoad.
+.org 08060DB2h
+.area 0Fh, 0
+    mov r0, #1
+    pop r1
+    bx  r1
+.endarea
+
+; Change QuarantineBayHornoad Palette
+.org SpritePalettePtrs + QuarantineBayHornoad_Id * 4
+    .dw 085A1E58h ; Unused Palette from Opening, RoS Themed Hornoad
```

Comment:
Similar to above, the abbreciation RoS is not quite clear here without further context.